client/lib/faux-plugins.js

```diff
@@ -207,6 +209,18 @@ module.exports = function(roomName) {
       }
     })
 
+    var parseYoutubeTime = function(time) {
+      var timeReg = /([0-9]+h)?([0-9]+m)?([0-9]+s?)?/
+      var match = time.match(timeReg)
+      if (!match) {
+        return 0
+      }
+      var hours = parseInt(match[1] || 0)
```

Comment:
These calls should probably specify base 10: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#Syntax